src/Reflectivity/Object.extension.st

```diff
@@ -17,3 +17,91 @@ Object >> asMethodConstant [
 	constNode link: link.
 	^const
 ]
+
+{ #category : #'*Reflectivity' }
+Object >> intanceSpecificMetaLinksAvailable [
+	^true
+]
+
+{ #category : #'*Reflectivity' }
+Object >> link: aMetaLink toAST: aNode [
```

Comment:
Can you comment why methods on Object are required?

Reply:
In most cases you delegate to target objects which is good. But why this Object methods are good idea?

Reply:
I see. I misunderstood it originally.
So anObject here is where metalink will be active. For other receivers metalink will be not executed for given ast-node. Right?

Reply:
Yes :)